### PR TITLE
Style fixes (line length, extra spaces, newline).

### PR DESCRIPTION
**Pull request by zoq**
Some minor style fixes.

### CODE REVIEW

**Review by shrit — APPROVED** [review]
Looks great to me, thanks for the fixes.

File: src/mlpack/bindings/python/print_class_defn.hpp [hunk 1]
```diff
@@ -129,11 +129,14 @@ void PrintClassDefn(
   std::cout << std::endl;
   std::cout << "  def get_cpp_params(self, return_str=False):" << std::endl;
   std::cout << "    params = self._get_cpp_params()" << std::endl;
-  std::cout << "    return process_params_out(self, params, return_str=return_str)" << std::endl;
+  std::cout << "    return process_params_out(self, params, "
+    << "return_str=return_str)" << std::endl;
```
Comment (rcurtin) [inline src/mlpack/bindings/python/print_class_defn.hpp:133]:
```suggestion
      << "return_str=return_str)" << std::endl;
```
Should be a double indent on the wrapped line I suppose?

File: src/mlpack/bindings/python/print_class_defn.hpp [hunk 2]
```diff
@@ -129,11 +129,14 @@ void PrintClassDefn(
...
   std::cout << std::endl;
   std::cout << "  def set_cpp_params(self, params_dic):" << std::endl;
-  std::cout << "    params_str = process_params_in(self, params_dic)" << std::endl;
-  std::cout << "    self._set_cpp_params(params_str.encode(\"utf-8\"))" << std::endl;
+  std::cout << "    params_str = process_params_in(self, params_dic)"
+    << std::endl;
```
Comment (rcurtin) [inline src/mlpack/bindings/python/print_class_defn.hpp:137]:
```suggestion
      << std::endl;
```

File: src/mlpack/bindings/python/print_class_defn.hpp [hunk 3]
```diff
@@ -129,11 +129,14 @@ void PrintClassDefn(
...
+  std::cout << "    self._set_cpp_params(params_str.encode(\"utf-8\"))"
+    << std::endl;
```
Comment (rcurtin) [inline src/mlpack/bindings/python/print_class_defn.hpp:139]:
```suggestion
      << std::endl;
```

File: src/mlpack/bindings/python/print_pyx.cpp [hunk 4]
```diff
@@ -80,8 +80,10 @@ void PrintPYX(const util::BindingDetails& doc,
   cout << "from io cimport EnableVerbose, DisableVerbose, DisableBacktrace, "
       << "ResetTimers, EnableTimers" << endl;
   cout << "from matrix_utils import to_matrix, to_matrix_with_info" << endl;
-  cout << "from preprocess_json_params import process_params_out, process_params_in" << endl;
-  cout << "from serialization cimport SerializeIn, SerializeOut, SerializeOutJSON, SerializeInJSON" << endl;
+  cout << "from preprocess_json_params import process_params_out, "
+    << "process_params_in" << endl;
```
Comment (rcurtin) [inline src/mlpack/bindings/python/print_pyx.cpp:84]:
```suggestion
      << "process_params_in" << endl;
```

**Review by rcurtin — COMMENTED** [review]
Just a couple small comments---everything else looks great to me. :+1:

**Review by rcurtin — APPROVED** [review]
Awesome, looks good to me. :+1:

**Pull request merged**